CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## 2.3.1 / 2023-07-19
```

Comment:
The new release should be 2.4.0 because it has new features and enhancement

Reply:
fixed